pkgs/games/dwarf-fortress/wrapper/default.nix

```diff
@@ -31,10 +31,12 @@
 }:
 
 let
-  dfhack_ = dfhack.override {
+  dfhack' = dfhack.override {
     inherit enableStoneSense;
   };
 
+  enableTWBT' = enableTWBT && (twbt != null);
```

Comment:
Why should twbt be null?

Reply:
TWBT doesn't even compile for version 50, and it's not needed anyway, so we shouldn't put it in the wrapper. I've improved the condition to check for the dfVersion as a passthru.